src/amuse/community/sakura/ggf/old_test_sakura.py

```diff
@@ -17,137 +17,137 @@ class TestSakuraInterface(TestWithMPI):
     def test01(self):
         if MODULES_MISSING:
             self.skip("Failed to import a module required for Sakura")
-        print "Test SakuraInterface initialization"
+        print("Test SakuraInterface initialization")
         instance = SakuraInterface()
-        self.assertEquals(0, instance.initialize_code())
-        self.assertEquals(0, instance.commit_parameters())
-        self.assertEquals(0, instance.cleanup_code())
+        self.assertEqual(0, instance.initialize_code())
```

Comment:
I am not sure  about the replacement of assertEquals with assertEqual - should check TestWithMPI

Reply:
I am not sure it matters here (see filename), but I have seen it elsewhere

Reply:
We may want to use this opportunity to do some cleanup... Also for ph4.

Reply:
> I am not sure about the replacement of assertEquals with assertEqual - should check TestWithMPI

This is a "preferred" naming change in Py3, but both should work in Py2 and Py3 iirc.

Reply:
https://docs.python.org/3/library/2to3.html?highlight=assertequal#2to3fixer-asserts

Reply:
ok, it does not matter, in test/amusetest.py:
```
    assertEqual = failUnlessEqual
    assertEquals = failUnlessEqual
```